notebooks/curation_api/python/create_dataset_from_local_file.ipynb

```diff
@@ -13,19 +13,19 @@
    "id": "cd7f8b6d",
    "metadata": {},
    "source": [
-    "The script in this notebook performs the upload of a local datafile to a given Collection (as identified by its Collection uuid), where the datafile becomes a Dataset accessible via the Data Portal UI.\n",
+    "The script in this notebook performs the upload of a local datafile to a given Collection (as identified by its Collection id), where the datafile becomes a Dataset accessible via the Data Portal UI.\n",
     "\n",
     "In order to use this script, you must...\n",
     "- have a Curation API key (obtained from upper-righthand dropdown in the Data Portal UI after logging in)\n",
-    "- know the uuid of the Collection to which you wish to upload the datafile (taken from `/collections/<collection_uuid>` in url path in Data Portal UI when viewing the Collection)\n",
+    "- know the id of the Collection to which you wish to upload the datafile (taken from `/collections/<collection_id>` in url path in Data Portal UI when viewing the Collection)\n",
     "\n",
     "_For **NEW** Dataset uploads_:\n",
     "- You must decide upon a string identifier (a \"curator tag\") to use to uniquely identify the resultant Dataset within its Collection. This tag **MUST NOT** be the tag of an existing Dataset within this Collection (otherwise the existing Dataset will be updated; read on below), and it **MUST NOT** conform to the uuid format. The tag suffix **MUST** be `.h5ad`.\n",
     "\n",
     "_For **replacing/updating** existing Datasets_:\n",
     "- Uploads to a curator tag for which there already exists a Dataset in the given Collection will result in the existing Dataset being replaced by the new Dataset created from the datafile that you \n",
     "are uploading.\n",
-    "- Alternatively, an existing dataset may be targeted for replacement by using the Dataset's Cellxgene uuid as the identifier when writing to S3. In this scenario, the uuid **MUST** have `.h5ad` appended as a suffix.\n",
+    "- Alternatively, an existing dataset may be targeted for replacement by using the Dataset's Cellxgene id as the identifier when writing to S3. In this scenario, the id **MUST** have `.h5ad` appended as a suffix.\n",
```

Comment:
I think we should use the new format: CELLxGENE

Reply:
I'll leave that as a separate PR for now.